example-apps/babel/src/authentication.js

```diff
@@ -9,7 +9,11 @@ const test = async (z /*, bundle */) => {
   // This method can return any truthy value to indicate the credentials are valid.
   // Raise an error to show
   if (response.status === 401) {
-    throw new Error('The username and/or password you supplied is incorrect');
+    throw new z.errors.Error(
+      'The username and/or password you supplied is incorrect',
+      'AuthenticationError',
```

Comment:
The second argument can be any code and as long as we're the only ones able to create troubleshooting docs, this isn't of much direct use to the developer.

However, the idea is/was that we'll grow a set of standard codes that we have generic troubleshooting docs on and ultimately, we'd to list the most common error codes we see for an app in its dashboard and maybe even allow partners to write their own troubleshooting docs for them.